src/tooltip.js

```diff
@@ -86,6 +86,10 @@ angular.module('cp-tooltip')
 					tooltipDisplayed = false;
 					if(tooltipEl) tooltipEl.remove();
 				}
+
+				function encodeHTML(str) {
+					return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+				}
```

Comment:
@blittle I am aware of that. I initially thought a naive solution would suffice in this case, since we didn't have any type of sanitization before. Do you think `ngSanitize` would be more appropriate?